spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/HttpServiceRegistrationBean.java

```diff
@@ -68,4 +72,84 @@ public HttpServiceRegistrationBean setRoute(@NotNull Route route) {
     public HttpServiceRegistrationBean setPathPattern(@NotNull String pathPattern) {
         return setRoute(Route.builder().path(pathPattern).build());
     }
+
+    @Override
+    public @NotNull Collection<Object> getExampleRequests() {
+        throw new UnsupportedOperationException();
```

Comment:
Perhaps we don't need to override this because it will return an empty list anyway?